src/Host/Client/Impl/BrokerServices/RemoteUriWebService.cs

```diff
@@ -55,7 +55,7 @@ public class RemoteUriWebService : IRemoteUriWebService {
             var remoteUri = GetRemoteUri(context.Request.Url, remoteBaseUrl);
             request.Headers.Add(CustomHttpHeaders.RTVSRequestedURL, remoteUri.ToString());
 
-            if (context.Request.InputStream.CanSeek && context.Request.InputStream.Length > 0) {
+            if (context.Request.ContentLength64 > 0) {
```

Comment:
`ContentLength64 > 0` only if the stream has content. Also, we need to do the copy only if the stream has content. The stream is still usable in the websocket context.